packages/TypeFormPopup/TypeFormPopup.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <a @click="onClickedPopup">
```

Comment:
If we put an `a` tag in the slot then we will have a link in a link which I'm not sure is something html valid.

Maybe we can just have `<slot @click="onClickedPopup" />`

Reply:
I have changed from `a` tag to `div` tag  because we can't use `slot` directly in `template`